gen_cert.py

```diff
@@ -1947,16 +1951,16 @@ def _generate_v4_certificate(
         # 0 1 - italic
         # 1 0 - bold
         # 1 1 - italic and bold
-        addMapping('OpenSans-Light', 0, 0, 'OpenSans-Light')
-        addMapping('OpenSans-Light', 1, 0, 'OpenSans-Bold')
-        addMapping('SourceSansPro-Regular', 0, 0, 'SourceSansPro-Regular')
-        addMapping('SourceSansPro-Regular', 1, 0, 'SourceSansPro-Bold')
-        addMapping('SourceSansPro-Regular', 1, 1, 'SourceSansPro-BoldItalic')
+        font_string = self.template_font_name + '-' + self.template_font_type
+        font_file = font_string + '.ttf'
+        addMapping(font_string, 0, 0, font_string)
+        addMapping(font_string, 1, 0, self.template_font_name+'-Italic')
+        addMapping(font_string, 1, 0, self.template_font_name+'-Bold')
+        addMapping(font_string, 1, 1, self.template_font_name+'-BoldItalic')
```

Comment:
Does pep8/pylint not complain about this?

We usually surround operators with whitespace.